### ISSUE TITLE
Failed to install 'AmphiNom' from GitHub

### ISSUE THREAD

**Issue by MaoJunZhong** [hcliedtke/defrostR#1]
HI:

 I had an error installing this R package，

    error: Failed to install 'AmphiNom' from GitHub:
  Does not appear to be an R package (no DESCRIPTION)
 
Here is my environment configuration
- Session info -----------------------------------------------------------------------------------------------------------------------------
 setting  value                         
 version  R version 4.0.2 (2020-06-22)  
 os       Windows 10 x64                
 system   x86_64, mingw32               
 ui       RStudio                       
 language (EN)                          
 collate  Chinese (Simplified)_China.936
 ctype    Chinese (Simplified)_China.936
 tz       Asia/Taipei                   
 date     2020-10-23                    

- Packages ---------------------------------------------------------------------------------------------------------------------------------
 package     * version date       lib source        
 assertthat    0.2.1   2019-03-21 [1] CRAN (R 4.0.2)
 backports     1.1.7   2020-05-13 [1] CRAN (R 4.0.0)
 callr         3.4.3   2020-03-28 [1] CRAN (R 4.0.2)
 cli           2.0.2   2020-02-28 [1] CRAN (R 4.0.2)
 colorspace    1.4-1   2019-03-18 [1] CRAN (R 4.0.2)
 crayon        1.3.4   2017-09-16 [1] CRAN (R 4.0.2)
 curl          4.3     2019-12-02 [1] CRAN (R 4.0.2)
 desc          1.2.0   2018-05-01 [1] CRAN (R 4.0.2)
 devtools    * 2.3.0   2020-04-10 [1] CRAN (R 4.0.2)
 digest        0.6.25  2020-02-23 [1] CRAN (R 4.0.2)
 dplyr         1.0.0   2020-05-29 [1] CRAN (R 4.0.2)

Any answer to a successful installation is meaningful

best wishes
MJ Zhong

**Comment by hcliedtke** [comment]
Hi there, sorry for the late reply, for some reason I was not notified of your comment. Are you still having problems installing?

**Comment by MaoJunZhong** [comment]
Yes, Liedtke, I tried GitHub installation and downloading to a local installation. The GitHub installation has the same problem as the above. It cannot be loaded after the local installation. It is not a valid R package. I use the source code directly, but it would be nice to be able to install it successfully.

best wishes
Zhong

**Comment by hcliedtke** [comment]
Hi. Thanks for brining this to my attention. I have made some changes to the vignette and to the .Rbuildignore files which might have caused problems. 

I have checked the build for MacOS (on my own system), Ubuntu (with Travis) and Windows (with win-builder). All were successful. Let me know if it still does not work for you and I will try to find a windows machine to test the installation from github.
